nh_eobs_mental_health/tests/nh_clinical_patient_observation_ews/test_is_refusal_in_effect.py

```diff
@@ -185,3 +194,28 @@ def test_child_two_refusal_episodes(self):
             self.ews_model.is_refusal_in_effect(
                 self.cr, self.uid, second_refusal, mode='child')
         )
+
+    def test_refused_then_pme(self):
+        """
+        Test that if the patient refuses their first observation then
+        refusal_in_effect is set to True using parent mode
+        """
+        self.test_utils_model.complete_obs(
+            clinical_risk_sample_data.REFUSED_DATA)
+        self.test_utils_model.start_pme()
+        refused = self.ews_model.is_refusal_in_effect(
+            self.cr, self.uid, self.test_utils_model.ews_activity.id)
```

Comment:
Just a reminder that in future you can use `test_utils_model.copy_instance_variables` in `setUp()` so you don't have to reference `self.test_utils_model.ews_activity.id`.

Don't think it's an issue here though because you only need one variable and it's not causing any line length problems.